src/modules/pm.c

```diff
@@ -108,6 +110,30 @@ static int on_new_suspend(sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bu
 
 /* Listener on logind session.Active for current session */
 static int on_session_change(UNUSED sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bus_error *ret_error) {
```

Comment:
I'd rename this function to `on_logind_changes` since we are now watching multiple logind properties.